packages/@aws-cdk/aws-rds/README.md

```diff
@@ -406,3 +406,50 @@ new rds.OptionGroup(stack, 'Options', {
   ],
 });
 ```
+
+### Serverless
+
+[Amazon Aurora Serverless]((https://aws.amazon.com/rds/aurora/serverless/)) is an on-demand, auto-scaling configuration for Amazon
+Aurora. The databse will automatically start up, shut down, and scale capacity
+up or down based on your application's needs. It enables you to run your database
+in the cloud without managing any database instances.
+
+The following example initializes an Aurora Serverless PostgreSql cluster.
+Aurora Serverless clusters can specify scaling properties which will be used to
+automatically scale the database cluster seamlessly based on the workload. 
+
+```ts
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as rds from '@aws-cdk/aws-rds';
+
+const vpc = new ec2.Vpc(this, 'myrdsvpc');
+
+const cluster = new rds.ServerlessDatabaseCluster(this, 'AnotherCluster', {
+  engine: rds.DatabaseClusterEngine.AURORA_POSTGRESQL,
+  parameterGroup: rds.ParameterGroup.fromParameterGroupName(this, 'ParameterGroup', 'default.aurora-postgresql10'),
```

Comment:
Why are we showing a default here...? I would just leave this out (or make this more interesting).

Reply:
the parameter group is not a default for `PostgreSql` - missing it will hit you with 
> Could not create a new ParameterGroup for an unversioned aurora-postgresql cluster engine. Please either use a versioned engine, or pass an explicit ParameterGroup when creating the cluster

Reply:
OK. I feel like this might be a usability issue, but let's leave this as-is for now.